spec/services/adapters/structs/income_details_spec.rb

```diff
@@ -15,9 +15,12 @@
           'ended_employment_within_three_months' => 'yes',
           'lost_job_in_custody' => 'yes',
           'date_job_lost' => Date.new(2023, 9, 1),
-          'income_above_threshold' => 'yes',
+          'income_above_threshold' => 'no',
```

Comment:
Is this a dramatic change? If so does there need to be a test for checking the various combinations of these fields?

Reply:
Not dramatic. I changed it to be consistent with the actual flow - the user would not be asked the subsequent questions (frozen assets, has property, or has savings) if their income was above the threshold. 

I changed it on the fixture in the schema, so this did fail a test and needed changing to no IIRC